test-one/src/index.js

```diff
@@ -1,14 +1,33 @@
 import { log } from "./helpers/logger";
 
-export const go = async () => {
-  // Your code should go here instead of the welcome message below. Create and modify additional files as necessary/appropriate.
+const isMultiple = (val, by) => {
+  return (val % by === 0)
+}
 
-  const welcomeMessage =
-    "Congrats, you're up and running! Welcome to the first moshtix code challenge! " +
-    "Please replace this message with your solution and feel free to add " +
-    "or modify other files as necessary. Good luck : )";
+export const BossHog = (val) => {
+  switch (true) {
```

Comment:
can you explain why you prefer `switch(true)` to an if statement?

Reply:
When thinking of my naive solution, I originally used an else-if statement. I questioned if this was the best way, and did research on what's faster. Numerous sources stated a switch out performed a if-else check. This is my concern mainly, as when I complete coding challenges a common error I see is it's not fast enough. So I try to think about efficiency often.